apps/examples/esp32_wifi_demo/Kconfig

```diff
@@ -0,0 +1,51 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+menuconfig EXAMPLES_ESP32WIFIDEMO
+    bool "ESP32 wifi example" 
+    depends on ESP32_WIFI_SUPPORT 
+    default n
+    help
+    esp32 wifi example,it directly call esp wifi driver interface.
+
+if EXAMPLES_ESP32WIFIDEMO
+
+config ESP_WIFI_MODE_STATION
+    bool "ESP32 wifi station test" 
+    default y
+    help
+    enable esp32 wifi station mode example, say N enable softap mode example
+
```

Comment:
plz check unnecessary blank line, it appears in several files.

Reply:
tks, fixed